Userland/Games/Chess/main.cpp

```diff
@@ -173,8 +173,8 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         chess_widget.set_time_control_increment(new_game_dialog->time_control_increment());
 
         Config::write_bool("Games"sv, "Chess"sv, "UnlimitedTimeControl"sv, new_game_dialog->unlimited_time_control());
-        Config::write_i32("Games"sv, "Chess"sv, "TimeControlSeconds"sv, new_game_dialog->time_control_seconds());
-        Config::write_i32("Games"sv, "Chess"sv, "TimeControlIncrement"sv, new_game_dialog->time_control_increment());
+        Config::write_u32("Games"sv, "Chess"sv, "TimeControlSeconds"sv, new_game_dialog->time_control_seconds());
```

Comment:
you are implicitly converting int to uint here. If you want to change this to u32, consider changing the time-related member variables.